manifests/j/Jidifudi/GT4T/8.40.230201/Jidifudi.GT4T.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Automatically updated by the winget bot at 2023/Feb/02
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Jidifudi.GT4T
+PackageVersion: 8.40.230201
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://gt4t.net/apps/gt4t_win_32_bit/GT4T_setup.exe
+  InstallerSha256: B47DF86CA711DA94446EDEF9CCD185E7848E0989C6763030FAE76FA0CC9126C1
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+- Architecture: x64
+  InstallerUrl: https://gt4t.net/apps/gt4t_win_64_bit/GT4T_setup.exe
+  InstallerSha256: 291DE2CB2C4B6C1117D397791C8EF277FEF791DB751C40513966611A7937E94B
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+- InstallerLocale: zh-CN
+  Architecture: x86
+  InstallerUrl: https://dl.gt4t.cn/downloads/apps/gt4t_win_32_bit/GT4T_setup.exe
+  InstallerSha256: B47DF86CA711DA94446EDEF9CCD185E7848E0989C6763030FAE76FA0CC9126C1
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+- InstallerLocale: zh-CN
+  Architecture: x64
+  InstallerUrl: https://dl.gt4t.cn/downloads/apps/gt4t_win_64_bit/GT4T_setup.exe
+  InstallerSha256: 291DE2CB2C4B6C1117D397791C8EF277FEF791DB751C40513966611A7937E94B
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+ManifestType: installer
+ManifestVersion: 1.2.0

```

Comment:
Should the manifest version be updated to 1.4.0 or is it fine to keep new manifests on 1.2.0 for now?